sdk/keyvault/azure-security-keyvault-jca/src/test/java/com/azure/security/keyvault/jca/implementation/utils/JsonConverterUtilTest.java

```diff
@@ -43,4 +53,44 @@ public void testToJson() {
         assertTrue(string.contains("cer"));
         assertTrue(string.contains("\"value\""));
     }
+
+    @Test
+    void testFromJsonWithInvalidTokenResponseBody() {
+        final String accessTokenBody = getAccessTokenBody("src/test/resources/aad/invalid-access-token-response.json");
```

Comment:
The invalid token can be retrieved from Microsoft Entra token point? 
1. If it can be retrieved from token endpoint, it's not a invalid token response.
2. If it can not be retrieved from token endpoint, it's not necessary to test it.

Reply:
Leave one for normal scenario test.